landa/landa_sales/report/landa_deliveries_and_payments_summaries/landa_deliveries_and_payments_summaries.json

```diff
@@ -0,0 +1,45 @@
+{
+ "add_total_row": 1,
+ "columns": [],
+ "creation": "2023-08-13 07:57:14.110476",
+ "disable_prepared_report": 1,
```

Comment:
```suggestion
 "disable_prepared_report": 0,
```

The report does take quite some time, if none or just one of the filters is set. So maybe it would make sense for this to be a prepared report? I think the effect would be that it doesn't use live data but calculates the result in advance and only recalculates on explicit request.

Reply:
I agree with that. I've only set the flag for debugging.

Reply:
This was a bad idea. The prepared report feature is not ready yet for our use case.